plugins/modules/aci_cloud_epg_selector.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Cindy Zhao <cizhao@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_epg_selector
+short_description: Manage Cloud Endpoint Selector (cloud:EPSelector)
+description:
+- Manage Cloud Endpoint Selector on Cisco Cloud ACI
+notes:
+- More information about the internal APIC class B(cloud:EPSelector) from
+  L(the APIC Management Information Model reference,https://developer.cisco.com/docs/apic-mim-ref/).
```

Comment:
I think we can move the notes section to the bottom of the description section for consistency